File: course.py
## course.py
class Course:
    def __init__(self, courseName, courseCode, creditHours):
        self._courseName = courseName
        self._courseCode = courseCode
        self._creditHours = creditHours
        self._students = []
    
    def enroll_student(self, student):
        self._students.append(student)
    
    def get_enrolled_students(self):
        return [student.get_name() for student in self._students]

File: src/models/Course.java
package models;
import java.util.*;

public class Course {
    private String courseName;
    private String courseCode;
    private int creditHours;
    private List<Student> students;

    public Course(String courseName, String courseCode, int creditHours) {
        this.courseName = courseName;
        this.courseCode = courseCode;
        this.creditHours = creditHours;
        this.students = new ArrayList<>();
    }

    public void enrollStudent(Student student) {
        students.add(student);
    }

    public List<Student> getEnrolledStudents() {
        return students;
    }
}
    